cpp/ql/lib/semmle/code/cpp/metrics/MetricFile.qll

```diff
@@ -209,7 +209,7 @@ private predicate aClassFile(Class c, File file) { c.getDefinitionLocation().get
 
 pragma[noopt]
 private predicate dependsOnFileSimple(MetricFile source, MetricFile dest) {
-  // class derives from classs
+  // class derives from class
```

Comment:
Did it mean this:
```suggestion
  // class derives from another class
```

That's closer to the comment below

Reply:
Yes I think that's accurate (and clearer).

Reply:
Works for me too.